contentcuration/contentcuration/frontend/shared/views/MarkdownEditor/plugins/registerCustomMarkdownField.js

```diff
@@ -0,0 +1,33 @@
+import Vue from 'vue';
+import vueCustomElement from 'vue-custom-element';
+
```

Comment:
This file contains some common logic for `MarkdownImageField` and `MarkdownFormulaField` custom elements